cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/OrderLineDTO.java

```diff
@@ -1,24 +1,22 @@
 package com.cloudesire.platform.apiclient.dto.model.dto;
 
+import com.cloudesire.platform.apiclient.dto.model.enums.LineType;
+import com.cloudesire.platform.apiclient.dto.model.interfaces.Line;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.cloudesire.platform.apiclient.dto.model.enums.LineType;
-import com.cloudesire.platform.apiclient.dto.model.interfaces.Line;
 import com.liberologico.cloudesire.common.MathConfiguration;
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import org.hibernate.validator.constraints.Length;
 
 import javax.validation.Valid;
-import javax.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import java.util.Objects;
 
 @ApiModel( description = "A billed unit" )
 public class OrderLineDTO implements Line
 {
-    @NotNull
```

Comment:
the descendents of this class are used for custom invoices/recurring costs, leave the NotNull